runtime/compiler/z/codegen/J9TreeEvaluator.cpp

```diff
@@ -7378,7 +7381,12 @@ reservationLockEnter(TR::Node *node, int32_t lwOffset, TR::Register *objectClass
       cg->generateDebugCounter("LockEnt/LR/CASSuccessfull", 1, TR::DebugCounter::Undetermined);
 
       // Lock is acquired successfully
+#if defined(TR_TARGET_64BIT) && (JAVA_SPEC_VERSION >= 19)
+      TR::LabelSymbol *incrementOwnedMonitorCountLabel = generateLabelSymbol(cg);
+      generateS390BranchInstruction(cg, TR::InstOpCode::BRC, TR::InstOpCode::COND_BRC, node, incrementOwnedMonitorCountLabel);
+#else
       generateS390BranchInstruction(cg, TR::InstOpCode::BRC, TR::InstOpCode::COND_BRC, node, helperReturnOOLLabel);
+#endif
```

Comment:
While you are here, do you mind fix the typo on the DebugCounter right above this section?  `CASSuccessfull` has an extra `l` at the end.

Reply:
Updated the comment in the debug counter